pathfinder/model/channel_view.py

```diff
@@ -1,26 +1,26 @@
 from enum import Enum
 
 from eth_utils import is_checksum_address
-from raiden_libs.types import Address, ChannelIdentifier
 
 from pathfinder.config import DEFAULT_PERCENTAGE_FEE
+from raiden_libs.types import Address, ChannelIdentifier
 
 
 class ChannelView:
     """
     Unidirectional view of a bidirectional channel.
     """
     class State(Enum):
-        OPEN = 1,
-        SETTLING = 2,
+        OPEN = 1
+        SETTLING = 2
```

Comment:
Why did you remove these commas here?

Reply:
That was a mistake before, in enums no commas are needed.

Reply:
ah ok safe, thank you